bower.json

```diff
@@ -17,7 +17,16 @@
     "package.json"
   ],
   "dependencies": {
-    "purescript-tuples": "master",
-    "purescript-gen": "master"
+    "purescript-arrays": "master",
+    "purescript-gen": "master",
+    "purescript-lists": "master",
+    "purescript-quickcheck": "master",
+    "purescript-quickcheck-laws": "master",
+    "purescript-tuples": "master"
+  },
+  "devDependencies": {
+    "purescript-console": "master",
+    "purescript-effect": "master",
+    "purescript-psci-support": "master"
```

Comment:
I noticed a few dependencies listed in the spago.dhall file are missing from the bower.json file. Do we still need this file?

Reply:
We do need to preserve this file (in order to publish the library, and for Bower users). We can generate the Bower file from the Spago file, or we can manually add the missing dependencies.